ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -397,37 +418,39 @@ zipOverLedgerTables f l tables2 =
 -- Separate so that we can have a 'TableStuff' instance for 'Ticked1' without
 -- involving double-ticked types.
 class TableStuff l => TickedTableStuff (l :: LedgerStateKind) where
-  -- | NOTE: The 'SingI' constraint is here for the same reason its on
-  -- 'projectLedgerTables'
-  projectLedgerTablesTicked :: SingI mk => Ticked1 l mk  -> LedgerTables l mk
-  withLedgerTablesTicked    ::             Ticked1 l any -> LedgerTables l mk -> Ticked1 l mk
+  -- | NOTE: The 'IsApplyMapKind mk2' constraint is here for the same reason
+  -- it's on 'projectLedgerTables'
+  projectLedgerTablesTicked :: IsApplyMapKind mk => Ticked1 l mk  -> LedgerTables l mk
+  -- | NOTE: The 'IsApplyMapKind mk2' constraint is here for the same reason
```

Comment:
Dangling comment?

Reply:
No, but the `mk2` is a typo. Should be `mk`.